grid.cpp

```diff
@@ -353,6 +356,7 @@ void initializeGrids(
    volGrid.updateGhostCells();
    fsGridGhostTimer.stop();
    phiprof::Timer getFieldsTimer {"getFieldsFromFsGrid"};
+   computeCoupling(mpiGrid, cells, volGrid);
```

Comment:
This isn't needed - coupling is still valid from the balanceLoad call on line 338.